ggircs-app/app/components/Layout/PageRedirectHandler.tsx

```diff
@@ -39,7 +39,7 @@ const PageRedirectHandler: React.FunctionComponent<Props> = ({
         environment,
         sessionQuery,
         {}
-      );
+      )  as unknown as PageRedirectHandlerQueryResponse;
```

Comment:
not sure why this all of a sudden started being angry, but casting it to the expected type appears to have fixed the issue.
"Type error: Property 'session' does not exist on type 'RelayObservable<PageRedirectHandlerQueryResponse>'."